index.html

```diff
@@ -7,285 +7,406 @@
+    </section>
+    <section class="wraps">
+      <section class="work">
+        <article class="article">
           <div class="work-area">
-            <h3 class="recent">My Recent Works</h3>
-            
-          <p class="part1">
-          <p class="hr"><hr></li> 
-          <p><img class="image" src="Img Placeholder.png" alt="Yoga Picture"></p>
-          <p class="story">Multi-Post Stories</p>
-          <p class="daily">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-          </p>
-
-        <ul class="language">
-          <li class="cssx"> <span class="css-text">css</span></li>
-          <li class="html"> <span class="html-text">html</span></li>
-          <li class="bootstrap"> <span class="boot-text">bootstrap</span></li>
-          <li class="ruby"> <span class="ruby-text">Ruby</span></li>
-        </ul>
-      
-          <button type="button" class="projectlinkx">See Project</button>
+            <h3 class="recent"> My Recent Works</h3>
+            <p class="part1">
+              <img class="hr" src="Pictures/hrline.svg" alt="line">
+              <p><img class="image" src="Img Placeholder.png" alt="Yoga Picture"></p>
+              <p class="story">Multi-Post Stories</p>
+              <p class="daily">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            </p>
+            <ul class="languagex">
+              <li class="cssx">css</li>
+              <li class="html">html</p>
+              <li class="bootstrap">bootstrap</li>
+              <li class="ruby">Ruby</li>
+            </ul>
+            <button type="button" class="projectlinkx">See Project</button>
           </div>  
         </article>
-       </section>
-
-        <div class="wrapper">
-          <article  class="box1">
-            <div class="box-text1">
+      </section>
+      <div class="wrapper">
+        <article  class="box1">
+          <div class="box-text1">
             <h3 class="title">Profesional Art Printing Data</h3>
             <p class="content">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
             <ul class="language1">
               <li class="html1">html</li>
               <li class="bootstrap1">bootstrap</li>
               <li class="ruby1">Ruby</li>
             </ul>
-            <div class="project"><a class="projectlink" href="#">See Project</a></div> 
-          </ul> 
+            <div class="project"><a class="projectlink" href="#">See Project</a></div>
           </div>
-          </article>
-    
-          <article class="box2">
-            <div class="box-text2">
+        </article>
```

Comment:
- [ ] **[OPTIONAL]** While it was not requested by the previous reviewer, I would really appreciate it if you could adjust your cards' projects width to fit in the entire page. Here is how it looks like:

![image](https://user-images.githubusercontent.com/47700024/140326453-f082ead3-6ecd-4bba-b33e-1edd3404777b.png)

This change is not mandatory as it wasn't requested by the first reviewer, but beware that in future milestones it might block you from your approval.